dbt/task/generate.py

```diff
@@ -90,6 +91,57 @@ def unflatten(columns):
     return structured
 
 
+def incorporate_catalog_unique_ids(catalog, manifest):
+    to_return = catalog.copy()
+
+    for schema, tables in to_return.items():
+        for table_name, table_def in tables.items():
+            unique_id = manifest.get_unique_id_for_schema_and_table(
+                schema, table_name)
+
+            if unique_id is None:
+                warning = (
```

Comment:
I think this warning might be a little chattier than anticipated. I bet this will show up dozens of times for a typical development invocation of dbt.